spec/features/users/user_can_delete_an_answer_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+feature 'user can delete an answer' do
+  scenario 'user owns an answer and clicks "Delete Answer"' do
+    user = Fabricate(:user, id: 1, name: "we_the_best")
+    user.roles.create(name: 'registered_user')
```

Comment:
With the fabricator piece from @TheKaterTot last PR, I'm pretty sure line 6 is unnecessary as a fabricated user is generated with a role of registered user by default; I may be mistaken.